### DISCUSSION TITLE
캐릭터 코딩 업로드

### DISCUSSION THREAD

**Discussion by himies** [naverz/zepeto-studio-kor#1225] [5. Q&A]
# 
![001](https://user-images.githubusercontent.com/111993537/186440110-9c4d327b-3122-4925-91dc-9da5c53f0d10.JPG)
#
아무리 해도 제 캐릭터가 업로드 되지 않고 기본 캐릭터로 업로드 되는데 무슨 문제가 있는걸까요?

**Comment by VontineDev** [comment]
ZepetoPlayers.instance.CreatePlayerWithZepetoId("", "[himes]", new SpawnInfo(), true);

"[himes]" 부분을 자신의 아이디로 바꿔보세요. "yourid" 이렇게요.

**Comment by zlmrzlo** [comment]
https://studio.zepeto.me/guides/zepetoplayer
이 부분 참고해보시면 좋을 것 같습니다.
